src/layers/NetzkartePointLayer/NetzkartePointLayer.js

```diff
@@ -0,0 +1,158 @@
+import qs from 'querystring';
+import VectorLayer from 'react-spatial/layers/VectorLayer';
+import OLVectorLayer from 'ol/layer/Vector';
+import OLVectorSource from 'ol/source/Vector';
+import OLGeoJSON from 'ol/format/GeoJSON';
+import { bbox as OLBboxStrategy } from 'ol/loadingstrategy';
+import { Style as OLStyle, Circle as OLCircle, Fill as OLFill } from 'ol/style';
+import CONF from '../../config/appConfig';
+import layerHelper from '../layerHelper';
+
+class NetzkartePointLayer extends VectorLayer {
+  constructor(options = {}) {
+    let name = 'Stationen';
+    let key = 'ch.sbb.stationen';
+
+    if (options.showAirports) {
+      name = 'Flughäfen';
+      key = 'ch.sbb.flughafen';
+    }
+
+    const vectorSource = new OLVectorSource({
+      format: new OLGeoJSON(),
+      ...(options.useBboxStrategy && { strategy: OLBboxStrategy }),
+    });
+
+    const olLayer = new OLVectorLayer({
+      source: vectorSource,
+    });
+
+    super({
+      ...options,
+      name,
+      key,
+      olLayer,
+      radioGroup: 'stationen',
+    });
+
+    // Options
+    this.showAirports = !!options.showAirports;
+    this.useBboxStrategy = !!options.useBboxStrategy;
+
+    // Bindings
+    this.airportStyle = this.airportStyle.bind(this);
+    this.defaultStyle = this.defaultStyle.bind(this);
+    this.loader = this.loader.bind(this);
+
+    // Style
+    this.netzkarteStyleCache = {};
+    olLayer.setStyle(
+      options.showAirports ? this.airportStyle : this.defaultStyle,
+    );
+
+    // Url
+    this.url = `${CONF.geoserverUrl}?service=WFS&version=1.0.0&request=GetFeature&`;
+    this.urlParams = {
+      typeName: options.showAirports
+        ? 'trafimage:netzkarte_airport_point'
+        : 'trafimage:netzkarte_point',
+    };
+
+    // Set loader after binding
+    this.vectorSource = vectorSource;
```

Comment:
I wouldn't use `vectorSource` as a class parameter. You can always do `this.olLayer.getSource()` which i find more explicit.